client/src/components/Header/index.js

```diff
@@ -22,8 +22,8 @@ const Header = () => {
             <Nav.Link className="header-links" as={Link} to="/myparks"
             >My Parks
             </Nav.Link>
-            <Nav.Link className="header-links" href="#link">
-              Get Pass
+            <Nav.Link className="header-links" href="/donation">
+              Donation
```

Comment:
Update this to, `<Nav.Link className=className="header-links" as={Link} to="/donation"> `  Using href will cause a re-render.  In react, only use href when you are linking to a site outside the application.

Reply:
sounds good, thanks!